js/foundation.abide.js

```diff
@@ -92,6 +102,14 @@ class Abide extends Plugin {
     this._init();
   }
 
+  /**
+   * Enables (true) or disables (false) the whole validation
+   * @param {Boolean} enable
+   */
+  enableValidation(enable) {
```

Comment:
Choose one:

```js
myform.enableValidation();
myform.disableValidation();
```

```js
myform.setValidation(true);
myform.setValidation(false);
```

```js
myform.isValidationEnabled = true;
myform.isValidationEnabled = false;
```

Reply:
I go with the first (already pushed it)

Initially I thought about naming it `toggleValidation` (same as `setValidation`) but didn't like it because it's no clear enough imo.
Two individual methods sound good to me.